lib/bucky/core/test_core/test_class_generator.rb

```diff
@@ -16,7 +16,13 @@ module TestClassGeneratorHelper
         def add_test_procedure(procedures)
           procedures.each.with_index(1) do |procedure, index|
             procedure[:proc] ||= ''.dup
-            puts procedure[:proc].insert(0, "  #{index}:") if procedure.key?(:proc)
```

Comment:
remove procedure.key?(:proc)
because that it set '' as default it has not procedure[:proc]